cas-management-webapp/src/main/java/org/jasig/cas/services/web/beans/RegisteredServiceEditBean.java

```diff
@@ -82,35 +83,80 @@
  * @since 4.1
  */
 public final class RegisteredServiceEditBean implements Serializable {
+    /**
+     * The constant serialVersionUID.
+     */
     private static final long serialVersionUID = 4882440567964605644L;
 
+    /**
```

Comment:
Seeing all these new useless Javadocs, I start thinking that our rules are too restrictive: what's the value of *The constant LOGGER.* Javadoc? Same question for *The Form data.*, *The Service data.*, *The Status.*...